src/cmd/manage_nsfs.js

```diff
@@ -227,17 +230,20 @@ async function verify_bucket_owner(bucket_owner) {
             const data = await get_config_data(full_path, show_secrets);
             if (data.email === bucket_owner) {
                 is_bucket_owner_exist = true;
+                is_allow_bucket_creation = data.allow_bucket_creation;
```

Comment:
@shirady can we use `break` here to exit the loop?

Reply:
No, you cannot use a `break` in a map.

> Array .map and .forEach are designed to never be stopped. Therefore, you cannot break out of the loop when using those methods.

([link](https://allaboutcode.medium.com/javascript-how-to-break-statement-in-an-array-map-method-9861b39957a4))

We are using it as a temporary solution to go over the elements with concurrency.
This implementation will be changed once we use account name (it will be with a lookup on the accounts names) - please see issue #7766.